rules.d/60-persistent-storage.rules.in

```diff
@@ -37,14 +37,22 @@ KERNEL=="nvme*[0-9]n*[0-9]", ENV{DEVTYPE}=="disk", ATTRS{serial}=="?*", ENV{ID_S
 KERNEL=="nvme*[0-9]n*[0-9]", ENV{DEVTYPE}=="disk", ATTRS{wwid}=="?*", ENV{ID_WWN}="$attr{wwid}"
 KERNEL=="nvme*[0-9]n*[0-9]", ENV{DEVTYPE}=="disk", ATTRS{model}=="?*", ENV{ID_MODEL}="$attr{model}"
 KERNEL=="nvme*[0-9]n*[0-9]", ENV{DEVTYPE}=="disk", ATTRS{firmware_rev}=="?*", ENV{ID_REVISION}="$attr{firmware_rev}"
+KERNEL=="nvme*[0-9]n*[0-9]", ENV{DEVTYPE}=="disk", ATTRS{nsid}=="?*", ENV{ID_NSID}="$attr{nsid}"
+# obsolete symlink that might get overridden on adding a new nvme controller, kept for backward compatibility
 KERNEL=="nvme*[0-9]n*[0-9]", ENV{DEVTYPE}=="disk", ENV{ID_MODEL}=="?*", ENV{ID_SERIAL_SHORT}=="?*", \
   OPTIONS="string_escape=replace", ENV{ID_SERIAL}="$env{ID_MODEL}_$env{ID_SERIAL_SHORT}", SYMLINK+="disk/by-id/nvme-$env{ID_SERIAL}"
+KERNEL=="nvme*[0-9]n*[0-9]", ENV{DEVTYPE}=="disk", ENV{ID_MODEL}=="?*", ENV{ID_SERIAL_SHORT}=="?*", ENV{ID_NSID}=="?*"\
```

Comment:
Missing `,` before the line continuation `\`

Reply:
@ldv-alt Interestingly, the failure is not detected by 'udevadm verify'.

Reply:
@yuwata Since `,` is treated as whitespace and there is a whitespace symbol after `,`, the line without `,` remains syntactically correct, even though it looks odd.

Reply:
More than that, the udev parser accepts the following syntax:
```
ENV{ID_NSID}!=""OPTIONS="string_escape=replace"
```
because closing `"` terminates the token, so even whitespace is not required by the parser.

Reply:
Ah, yeah, I remembered that few years ago, I tried to add warning about missing comma (and/or white space), then IIRC @keszybz said no.

I think `udevadm verify` should warn that, but `udevd` do not.

Reply:
> Missing `,` before the line continuation `\`

Argh, sorry, I didn't have time to test my latest changes yet.
So I just submitted but didn't announce it.